aws/eks-helm/metrics-server/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+## [0.1.28]() (2024-12-27)
+### Features
+* Init `eks rbac` module
```

Comment:
For clarification, could you explain the necessity of a sub-module for RBAC? The existing chart configuration already defaults `rbac.create` to true.